webhooks-extension/test/cleanup.sh

```diff
@@ -9,8 +9,11 @@ source ${test_dir}/util.sh
 if [ $DASHBOARD_INSTALL_NS != "default" ]; then
   kubectl delete ns ${DASHBOARD_INSTALL_NS}
 else
-  kubectl delete deployment tekton-dashboard -n $DASHBOARD_INSTALL_NS
+  kubectl delete deployment tekton-dashboard -n default
+  kubectl delete deployment webhooks-extension -n default
```

Comment:
Same as above.

Reply:
As above, ns is explicitly default. The whole cleanup script is not called from anywhere, and not documented: it was for my convenience only.